nilearn/masking.py

```diff
@@ -513,6 +513,137 @@ def compute_multi_background_mask(data_imgs, border_size=2, upper_cutoff=0.85,
     return mask
 
 
+def compute_gray_matter_mask(target_img, threshold=.5,
+                             connected=True, opening=2, memory=None,
+                             verbose=0):
+    """ Compute a mask corresponding to the gray matter part of the brain.
+    The gray matter part is calculated through the resampling of MNI152
+    template Gray Matter mask onto the image
+
+    Parameters
+    ----------
+    target_img: Niimg-like object
+        See http://nilearn.github.io/manipulating_images/input_output.html
+        Images used to compute the mask. 3D and 4D images are accepted.
+        Only the shape and affine of target_img will be used here.
+
+    threshold: float, optional
```

Comment:
Please add the default here.

Reply:
I don't see the need of threshold ? The values contained in the brain mask are [0, 1]

Reply:
Because we use `continuous` interpolation scheme when doing the resampling, the values of `resampled_template.get_data()` will have arbitrary values, not only zeros and ones, that's why we use this threshold